includes/functions.php

```diff
@@ -3963,4 +3963,54 @@ function pmpro_get_ip() {
 	$client_ip = preg_replace( '/[^0-9a-fA-F:., ]/', '', $client_ip );
 	
 	return $client_ip;
+}
+
+/**
+ * Determines if this order can be refunded
+ * @param  object $order The order that we want to refund
+ * @return bool Returns a bool value based on if the order can be refunded
+ */
+function pmpro_allowed_refunds( $order ) { 
+
+	if( empty( $order ) || empty( $order->gateway ) ) {
+		return false;
+	}
+
+	if( 
+		in_array( $order->status, array( 'success' ) ) && //Only successfully paid orders
+		in_array( $order->gateway, array( 'stripe', 'paypal-express' ) )  //Only apply to these gateways
```

Comment:
WP PHPCS says this should always have third parameter set to `true`.

```suggestion
		in_array( $order->gateway, array( 'stripe', 'paypal-express' ), true )  //Only apply to these gateways
```